python-avd/pyavd/_eos_designs/structured_config/core_interfaces_and_l3_edge/ethernet_interfaces.py

```diff
@@ -30,9 +30,10 @@ def ethernet_interfaces(self: AvdStructuredConfigCoreInterfacesAndL3Edge) -> lis
         for p2p_link in self._filtered_p2p_links:
             if p2p_link["data"]["port_channel_id"] is None:
                 # Ethernet interface
+                context = f"{self.data_model}"
```

Comment:
cant we use the self.data_model directly inside the self._get_ethernet_cfg method

Reply:
No. It is not possible.

Reply:
It worked. Thanks.